blueflood-core/src/main/java/com/rackspacecloud/blueflood/utils/CalendarClock.java

```diff
@@ -0,0 +1,21 @@
+package com.rackspacecloud.blueflood.utils;
+
+import org.joda.time.Instant;
+
+import java.util.Calendar;
+
+public class CalendarClock implements Clock {
```

Comment:
I actually don't know why we call ```Calendar.getInstance().getTimeInMillis()```. That was the first question I asked when I saw that code.